packages/suite/src/views/wallet/transactions/components/TokenList/index.tsx

```diff
@@ -85,16 +85,19 @@ const TokenList = ({ tokens, explorerUrl, isTestnet }: Props) => {
                     <Fragment key={t.address}>
                         <Col>
                             <TokenName>
-                                <TokenSymbol>
```

Comment:
I would prefer to exclude this token from the list completely (whole row)
if it doesnt have a name/shortcut/balance then its not a valid token